daemon/state.go

```diff
@@ -104,7 +104,10 @@ func (d *Daemon) restoreOldEndpoints(dir string, clean bool) (*endpointRestoreSt
 				}
 			}
 
-			if _, err := netlink.LinkByName(ep.IfName); err != nil {
+			// FIXME: ignore the check for ipvlan slave because it requires
+			//        entering container netns which is not always accessible
+			//        (e.g. in k8s case "/proc" has to be bind mounted)
+			if _, err := netlink.LinkByName(ep.IfName); option.Config.DatapathMode != option.DatapathModeIpvlan && err != nil {
```

Comment:
Should we make this a small helper here and in ipvlan case check for existence of the EP's tail call map (otherwise we fail later on when doing map graft)?